src/Stratis.Bitcoin.Features.Api/Program.cs

```diff
@@ -20,7 +20,7 @@ public static IWebHost Initialize(IEnumerable<ServiceDescriptor> services, FullN
                 .UseUrls(fullNode.Settings.ApiUri.ToString())
                 .ConfigureServices(collection =>
                 {
-                    if (services == null || fullNode == null)
+                    if (services == null)
```

Comment:
Nice :-)
Maybe we should add a Guard like this:
`Guard.NotNull(fullNode, nameof(fullNode));`